src/api/utils/setting.py

```diff
@@ -0,0 +1,24 @@
+import os
+from typing import Any
+
+
+def setting(name: str, default: Any = None, *, required: bool = False) -> str:
+    """
+    Get a setting from the environment.
+    Default and Required cannot be set at the same time.
+
+    :param name: Environment variable name
+    :param default: Value to use if variable is not assigned in environment
+    :param required: If the variable is not assigned in teh environment, raise an error
+    :return:
+    """
+
+    if default and required:
+        raise EnvironmentError(
+            f"Environment variable cannot have a default value and be required: {name} "
+        )
+
+    env_var = os.getenv(name, default)
+    if required and env_var is None:
+        raise EnvironmentError(f'Missing environment variable: {name}')
+    return env_var
```

Comment:
Can we make this `return str(env_var)`? This is because default is of the `typing.Any` type, so if the environment variable is not found, `env_var` will be set to `default` and hence _may_ not be a string. Typecasting the return to a string will ensure that all values coming out of the `setting` will be a string no matter what.

Reply:
Example:
![image](https://user-images.githubusercontent.com/57634280/115115168-715e5480-9fb0-11eb-91d8-f2cc1c1d1c08.png)

Reply:
On it now, good suggestion.